docs/r2/tensorboard_quickstart.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>from_logits should be False as you are using softmax in the last layer</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/1870/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Good catch!